### PR TITLE
reducing padding above lists

### PR DESCRIPTION
**Pull request by samjewell**
## This

screen shot 2015-04-07 at 11 30 50

### CODE REVIEW

**Comment by moyni** [comment]
This solves my problem! Thank you. Then I can change top margin for the lists that need space above, right?
& is there a lazy way to see whether there are lists on the splash pages other than the jobs ones?

**Comment by samjewell** [comment]
Hey @moyni you're welcome :-)
Guess you'll want to do a "find in files" type of search, to search the whole 'pages' folder for <ul> tags? In sublime I use ` find > find in files ` or cmd-shift-F. Or even better - right click the "pages" folder that you want to search in, and then choose "find in folder" in there...

**Comment by moyni** [comment]
@samjewell can you have a look at this, please?

<ul> preceded by <p> is fine
<ul> preceded by <h> needed top margin added
<ul> embedded in another list needed margins cleared.

File: app/pages/fr/legal/activites-restreintes/index.html [hunk 1]
```diff
@@ -39,7 +39,7 @@ <h2 class="u-text-heading-light u-text-m u-color-heading">Activités restreintes
         <ul class="list">
           <li>
             Utiliser les services de GoCardless d’une quelconque façon ou en connexion avec toute activité qui :
-            <ul class="list">
+            <ul class="list u-margin-Tn">
```
Comment (samjewell) [inline app/pages/fr/legal/activites-restreintes/index.html]: Codez look great in general :-D ... I'll take a look in the browser now too.
Guessing this line, and the other ones below this on in this PR should be `Tm` not `Tn`?

**Comment by samjewell** [comment]
For anyone who's wondering, bullets look like this in the jobs pages now:
screen shot 2015-04-07 at 17 01 09

screen shot 2015-04-07 at 17 01 38

File: app/pages/fr/legal/activites-restreintes/index.html (hunk 1 above)
Comment (moyni) [inline app/pages/fr/legal/activites-restreintes/index.html]: `Tm` looked slightly strange in the sublists, `Tn` looked better I think.

**Comment by samjewell** [comment]
@moyni looks good to me :+1: , after the `Tn` thing...

File: app/pages/fr/legal/activites-restreintes/index.html (hunk 1 above)
Comment (samjewell) [inline app/pages/fr/legal/activites-restreintes/index.html]: Woah, I didn't even realise `Tn` was a thing, and that we had it in the repo! Well great stuff, awesome :racehorse:

**Pull request merged**